gatsby-config.js

```diff
@@ -702,6 +702,32 @@ module.exports = {
                     }
 
                 ]
+            },{
+                title: "Migration",
+                path: "/resources/migration",
+                pages: [{
+                        title: "Android",
+                        path: "/resources/migration/android",
+                        pages: [{
+                                title: "Migrate to Android 2.x SDKs",
+                                path: "/resources/migration/android/migrate-to-android-2x"
+                            }
+                        ]
+                    },
+                    {
+                        title: "iOS",
+                        path: "/resources/migration/ios",
+                        pages: [{
+                                title: "Migrate to iOS 3.x SDKs",
+                                path: "/resources/migration/ios/migrate-to-ios-3x"
+                            },
+                            {
+                                title: "Migrate to iOS 4.x SDKs",
+                                path: "/resources/migration/ios/migrate-to-ios-4x"
+                            }
+                        ]
+                    }
+                ]
```

Comment:
Since this is a sub-page to resource, this can just be listed as a subpage under resources (so, similarly to `                        path: "/resources/upgrade-platform-sdks",`).

Reply:
@calebk1 I wanted to keep this section at the top level similar to `User Guides`. I placed the content in the `resources` directory, following the same approach used for 'User Guides'. Let me know if you need me to move it our of resources.